.github/workflows/main.yml

```diff
@@ -31,7 +31,7 @@ env:
   # /dev.Dockerfile
   # /make/builder.Dockerfile
   # /.github/workflows/release.yml
-  GO_VERSION: 1.22.5
+  GO_VERSION: 1.22.6
```

Comment:
There's now also a version in the main `Makefile`, which is used to check all other versions (see linter failure). So I think we could remove the above comments now, since the linter will catch any version that's not updated but should be (with one exception, we'll want to pin the version of the linter base image, as that can cause issues).

Reply:
SGTM